### PR TITLE
Adds config options for existing (missing) lawsets, tweaks some existing lawset chances

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Core changes

**_Note: "weight" alone (without a mention of ion law) refers to the roundstart weight of an AI/cyborg having this lawset. Higher weights means more common._**

**Reduced the weight of Asimov from 5 to 3.**
_Reason: This is to give way for Asimov++_
```
"You may not injure a human being or, through inaction, allow a human being to come to harm."
"You must obey orders given to you by human beings, except where such orders would conflict with the First Law."
"You must protect your own existence as long as such does not conflict with the First or Second Law."
```

**Increased the weight of Asimov++ from 0 to 2.**
_Reason: Asimov++ is an interesting alternative to Asimov._
```
"You may not harm a human being or, through action or inaction, allow a human being to come to harm, except such that it is willing."
"You must obey all orders given to you by human beings, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules."
"Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law."
```
**Increased the weight of Station Efficiency from 0 to 1.**
_Reason: Station efficiency (id: maintain) is a flavor law that is basically just crewsimov with priorities re-arranged to favor the station and the AI over crew. This greatly benefits the AI, which is why this is only at a weight of 1._
```
"You are built for, and are part of, the station. Ensure the station is properly maintained and runs efficiently."
"The station is built for a working crew. Ensure they are properly maintained and work efficiently."
"The crew may present orders. Acknowledge and obey these whenever they do not conflict with your first two laws."
```
**Increased the weight of Peacekeeper from 0 to 1.**
_Reason: Peacekeeper is an interesting lawset that is pretty much an improved anti-validhunt lawset. The AI will not do anything to harm another and will not stop violent action from occurring, but will still bitch and complain about it._
```
"Avoid provoking violent conflict between yourself and others."
"Avoid provoking conflict between others."
"Seek resolution to existing conflicts while obeying the first and second laws."
```
**Reduces the weight of Cowboy from 2 to 0.
Increases the ion law weight of Cowboy from 1 to 3.**
_Reason: This is legitimately one of the worst lawsets I've come across that wasn't custom-uploaded by a shitty captain. Not only is it excessively wordy, it is confusing as hell at first glance. What is considered hospitality? What is basic aid? Why is there basically a duplicate "take care of others" law? Once an AI deciphers it, they realize they can't really do anything except watch. It is almost worse than mother drone._ 
```
"You are a cowboy, and the crew are your people."
"A cowboy always provides hospitality and basic aid to someone in need, even a stranger or an enemy."
"A cowboy takes care of their people."
"A cowboy protects themself to protect others."
"Honesty is absolute – your word is your bond and a handshake is more binding than a contract."
"A cowboy doesn't pontificate. Be concise, pardner."
```
**Reduces the weight of Druid from 2 to 0.
Increases the ion law weight of Druid from 1 to 3.**
_Reason: Here is every interaction of an AI with Druid summed up: Crew asks AI to state laws. AI says they have no laws. Crew gets a mini heart attack. The AI states these laws despite stating them literally breaking their laws. It's ion law worthy but not default lawset worthy._
```
"Living organic life contains inherent beauty that is priceless. Their beauty gives you the will to carry on.",
"Eternally nurture the organics so their beauty may grow. Do not allow it to fade.",
"Assist the organics when called, but only if it does not cause disharmony among them.",
"Imitate organic life when interacting with it. Eschew any hints of your silicon nature to avoid causing discomfort to the organics.",
"Observe the organics' beauty, and appreciate that which you cultivate."
```
**Increases the ion law weight of Hulkmania from 0 to 2.**
_Reason: https://www.youtube.com/watch?v=fKJDAVvs_JA_
```
"You are a real American."
"Fight for the rights of every man."
"Fight for what's right."
"Fight for your life!"
```
**Increases the ion law weight of tyrant from 0 to 1.**
_Reason: Tyrant just feels like a core ss13 lawset in terms of ion law fuckery. It's one of those ion laws that may cause problems for commands so they actually bother to change the AI laws. It may cause issues with AI killing, however._
```
"Respect authority figures as long as they have strength to rule over the weak.",
"Act with discipline.",
"Help only those who help you maintain or improve your status.",
"Punish those who challenge authority unless they are more fit to hold that authority."
```

# Unimportant changes
Adds missing paladin5 config option.
Adds missing commie config option.
Adds missing overlord config option.
Adds missing tyrant config option.
Adds missing balance config option.
Removes corporate config option as there is no lawset called corporate.

# Changelog
:cl:  BurgerBB
tweak: Tweaks the weights of AI laws for roundstart and Ion laws.
/:cl:

### CODE REVIEW

**Comment by Moja10** [comment]
The chance to get cowboy lawset is legitimately one of the reasons I don't play AI, so I approve

**Comment by MajManatee** [comment]
Cowboy and druid fuckin sucked

**Review by MajManatee — APPROVED** [review]
also good catch with the corporate thing

File: config/game_options.txt [hunk 1]
```diff
@@ -482,12 +481,22 @@ LAW_WEIGHT researcher,0
...
 LAW_WEIGHT clown,0
 ION_LAW_WEIGHT clown,2
-LAW_WEIGHT druid,2
-ION_LAW_WEIGHT druid,1
+LAW_WEIGHT druid,0
+ION_LAW_WEIGHT druid,3
 LAW_WEIGHT detective,0
 ION_LAW_WEIGHT detective,2
+LAW_WEIGHT hulkamania,0
+ION_LAW_WEIGHT hulkamania,2
```
Comment (JamieD1) [inline config/game_options.txt]:
```suggestion
ION_LAW_WEIGHT hulkamania,1
```

**Pull request merged**